### PR TITLE
TST: Added test for json containing the key "last_status_change_at"

### LINKED ISSUE #32409: pandas1.0.1 has trouble with certain column names
#### Code Sample, a copy-pastable example if possible

```python

df = pd.read_json(file_path,lines=True)

TypeError: unhashable type: ‘dict’ .  `

```
#### Problem description

Pandas1.0.1 with python3.7 hits the above for json containing dicts containing the key "last_status_change_at" and possibly others, while pandas0.25.3 with python3.7 does not. I haven't checked yet with python3.6. 

#### Expected Output
no unhashableness
#### Output of ``pd.show_versions()``

<details>

INSTALLED VERSIONS (pandas 1.0.1 which hits error)
------------------
commit           : None
python           : 3.7.6.final.0
python-bits      : 64
OS               : Darwin
OS-release       : 18.7.0
machine          : x86_64
processor        : i386
byteorder        : little
LC_ALL           : en_US.UTF-8
LANG             : en_US.UTF-8
LOCALE           : en_US.UTF-8
pandas           : 1.0.1
numpy            : 1.18.1
pytz             : 2019.3
dateutil         : 2.8.1
pip              : 19.3.1
setuptools       : 42.0.2
Cython           : None
pytest           : None
hypothesis       : None
sphinx           : None
blosc            : None
feather          : None
xlsxwriter       : None
lxml.etree       : None
html5lib         : None
pymysql          : None
psycopg2         : None
jinja2           : None
IPython          : 7.12.0
pandas_datareader: None
bs4              : None
bottleneck       : None
fastparquet      : None
gcsfs            : None
lxml.etree       : None
matplotlib       : 3.1.3
numexpr          : None
odfpy            : None
openpyxl         : None
pandas_gbq       : None
pyarrow          : 0.16.0
pytables         : None
pytest           : None
pyxlsb           : None
s3fs             : None
scipy            : 1.4.1
sqlalchemy       : None
tables           : None
tabulate         : None
xarray           : None
xlrd             : None
xlwt             : None
xlsxwriter       : None
numba            : None

INSTALLED VERSIONS  (pandas0.25.3 which doesnt hit error)
------------------
commit           : None
python           : 3.7.6.final.0
python-bits      : 64
OS               : Darwin
OS-release       : 18.7.0
machine          : x86_64
processor        : i386
byteorder        : little
LC_ALL           : en_US.UTF-8
LANG             : en_US.UTF-8
LOCALE           : en_US.UTF-8
pandas           : 0.25.3
numpy            : 1.18.1
pytz             : 2019.3
dateutil         : 2.8.1
pip              : 19.3.1
setuptools       : 42.0.2
Cython           : None
pytest           : None
hypothesis       : None
sphinx           : None
blosc            : None
feather          : None
xlsxwriter       : None
lxml.etree       : None
html5lib         : None
pymysql          : None
psycopg2         : None
jinja2           : None
IPython          : 7.12.0
pandas_datareader: None
bs4              : None
bottleneck       : None
fastparquet      : None
gcsfs            : None
lxml.etree       : None
matplotlib       : 3.1.3
numexpr          : None
odfpy            : None
openpyxl         : None
pandas_gbq       : None
pyarrow          : 0.16.0
pytables         : None
s3fs             : None
scipy            : 1.4.1
sqlalchemy       : None
tables           : None
xarray           : None
xlrd             : None
xlwt             : None
xlsxwriter       : None
</details>

### PR DESCRIPTION
**Pull request by rivera-fernando**
- [x] closes #32409
- [x] tests added / passed
- [x] passes `black pandas`
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

**Review by mroeschke — CHANGES_REQUESTED** [review]
Could you move this test to `pandas/tests/frame/methods/test_describe.py`?

And structure the test similar to what is describe in our guide?

https://pandas.pydata.org/pandas-docs/stable/development/contributing.html#writing-tests

File: pandas/tests/frame/methods/test_describe.py [hunk 1]
```diff
@@ -298,3 +298,13 @@ def test_describe_percentiles_integer_idx(self):
             ],
         )
         tm.assert_frame_equal(result, expected)
+
+
+def test__unhashable_dict():
+    result = pd.DataFrame(
```
Comment (mroeschke) [inline pandas/tests/frame/methods/test_describe.py:304]: - Could you remove the extra `_` in `test__`
- Could you add the issue number as a comment?
- This test should test this behavior: https://github.com/pandas-dev/pandas/issues/32409#issuecomment-626423254

**Comment by jreback** [comment]
@rivera-fernando can you merge master and respond to comments

File: pandas/tests/frame/methods/test_describe.py [hunk 2]
```diff
@@ -298,3 +298,14 @@ def test_describe_percentiles_integer_idx(self):
             ],
         )
         tm.assert_frame_equal(result, expected)
+
+
+def test_unhashable_dict():
+    # GH#32409
+    result = pd.DataFrame(
```
Comment (MarcoGorelli) [inline pandas/tests/frame/methods/test_describe.py:305]: Thanks @rivera-fernando 

What's the difference between `result` and expected? Can you construct `result` using `.describe` (as in the original issue)?

**Review by MarcoGorelli — CHANGES_REQUESTED** [review]

**Comment by simonjayhawkins** [comment]
closing as stale

**Pull request closed without merging**